clouddrift/signal.py

```diff
@@ -10,7 +10,7 @@ def analytic_signal(
     x: np.ndarray | xr.DataArray,
     boundary: str = "mirror",
     time_axis: int = -1,
-) -> np.ndarray | tuple[np.ndarray, np.ndarray]:
+) -> np.ndarray:
```

Comment:
The output CAN be a tuple depending on the input. Why remove it?

Reply:
Got it, will update it back.